### PR TITLE
Better rendering for all core commands

### PR DESCRIPTION
**Pull request by nclsHart**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | no
| License       | MIT

Example with the install command:

### Before

wallabag-install-previous

### Now

wallabag-install-now

### CODE REVIEW

File: src/Wallabag/CoreBundle/Command/InstallCommand.php [hunk 1]
```diff
@@ -260,14 +247,9 @@ protected function setupAdmin()
...
-        $question = new Question('Username (default: wallabag) :', 'wallabag');
-        $user->setUsername($questionHelper->ask($this->defaultInput, $this->defaultOutput, $question));
-
-        $question = new Question('Password (default: wallabag) :', 'wallabag');
-        $user->setPlainPassword($questionHelper->ask($this->defaultInput, $this->defaultOutput, $question));
-
-        $question = new Question('Email:', '');
-        $user->setEmail($questionHelper->ask($this->defaultInput, $this->defaultOutput, $question));
+        $user->setUsername($this->io->ask('Username', 'wallabag'));
+        $user->setPlainPassword($this->io->askHidden('Password'));
```
Comment (nclsHart) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: Warning, this could be a potential BC break : when using this method, password can't have default value.

Let me know if you want me to revert this change.

Comment (nclsHart) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: As I feared this change breaks things but I have found a solution to keep the previous behavior.

**Comment by j0k3r** [comment]
Looking great!
